src/components/Popups/PopupHighlight.scss

```diff
@@ -0,0 +1,52 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba {
+    $popup-arrow-height: 3px;
+    $popup-arror-width: 4px;
+    $popup-content-bg: $bdl-gray;
+
+    .ba-PopupHighlight {
+        pointer-events: auto;
+
+        // Top-facing arrows
+        &[data-popper-placement*='bottom'] {
+            .ba-Popup-arrow {
+                top: -$popup-arrow-height;
+            }
+        }
+
+        .ba-Popup-arrow {
+            &::before {
+                border-top: $popup-arrow-height solid $popup-content-bg;
+                border-right: $popup-arror-width solid transparent;
+                border-left: $popup-arror-width solid transparent;
+            }
+
+            &::after {
+                display: none;
+            }
+        }
+
+        .ba-Popup-content {
+            background-color: $popup-content-bg;
+            border: none;
+            border-radius: 4px;
+            box-shadow: none;
+        }
+    }
+
+    .ba-PopupHighlight-button,
+    .ba-PopupHighlight-button:hover,
```

Comment:
Specifying each pseudo-class shouldn't be needed here, since we're not using BUIE styling.